ui/src/routes/_layout/admin/runs/index.tsx

```diff
@@ -141,7 +141,7 @@ const columns = [
     header: 'Run Status',
     cell: ({ row }) => (
       <Badge
-        className={`border ${getStatusBackgroundColor(row.original.status)} hover:bg-${getStatusBackgroundColor(row.original.status)}`}
+        className={`border ${getStatusBackgroundColor(row.original.status)}`}
```

Comment:
So I would rather add `pointer-events-none` to these places where `Badge` is used and keep the colors as-is.

Reply:
> So I would rather add `pointer-events-none` to these places where `Badge` is used and keep the colors as-is.

So remove them from get... color class helper functions and add them instead everywhere where a Badge is used?

Reply:
Exactly.

Reply:
> Exactly. 

So when you say "a bit more idiomatic" you really mean "blow up this tiny PR"? : 😏😏

Reply:
Sure you could include the `pointer-events-none` as a shared value in the helper function return if that makes more sense to you. The non-idiomatic part is specifying hover color when what we actually want is the color not to change.

Reply:
More options could be to create a wrapper component like `StatusBadge` that does it, or as it feels that we by-default don't want the badges to have hover color, just remove the hover colors from [here](https://github.com/eclipse-apoapsis/ort-server/blob/main/ui/src/components/ui/badge-variants.ts#L17-L23).

Reply:
I agree the last option would probably be best.

Reply:
Done and seems to be working.